dot/config.go

```diff
@@ -106,14 +106,26 @@ type CoreConfig struct {
 
 // RPCConfig is to marshal/unmarshal toml RPC config vars
 type RPCConfig struct {
-	Enabled    bool
-	External   bool
-	Port       uint32
-	Host       string
-	Modules    []string
-	WSPort     uint32
-	WS         bool
-	WSExternal bool
+	Enabled          bool
+	External         bool
+	Unsafe           bool
+	UnsafeExternal   bool
+	Port             uint32
+	Host             string
+	Modules          []string
+	WSPort           uint32
+	WS               bool
+	WSExternal       bool
+	WSUnsafe         bool
+	WSUnsafeExternal bool
+}
+
+func (r RPCConfig) IsRPCEnabled() bool {
```

Comment:
Can we make `RPCConfig` pointer? This will end up creating multiple copies.
```
func (r *RPCConfig) IsRPCEnabled() bool {
```

Reply:
done